.nvmrc

```diff
@@ -1 +0,0 @@
-v16.15.0
```

Comment:
You shouldn't delete files. This is required for cloudflare to deploy.

Reply:
odd. i don't recall deleting this. must have been during a revert or something. i shall put back